tests/common/test_transform.py

```diff
@@ -0,0 +1,78 @@
+from gemstone.common.transform import *
+from gemstone.generator import Const
+import magma
+import fault
+import tempfile
+
+
+def test_remove_simple():
```

Comment:
Good test. However, can we have the new generator be different? And then check that the original one has one semantics, and the one after `replace` has different?

Reply:
Added more complex tests.